src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1443,8 +1443,8 @@
   <data name="General_Repository.Text" xml:space="preserve">
     <value>GitHub repository</value>
   </data>
-  <data name="General_Version.Header" xml:space="preserve">
-    <value>Version</value>
+  <data name="General_VersionAndUpdate.Header" xml:space="preserve">
+    <value>Version &amp; update</value>
```

Comment:
While not a problem, I personally don't feel much extra value to add this.

Reply:
Sure. But for me it felt strange to have uodate settings under "Version" category.